ethicml/evaluators/evaluate_models.py

```diff
@@ -321,12 +323,14 @@ def _gather_metrics(
                 for k, v in model.get_hyperparameters().items()
             }
 
-            df_row: Dict[str, Union[str, float]] = {
+            seed = predictions.info["model_seed"]
```

Comment:
I just looked at this. I guess we'd want `model_seed` and `data_seed`, and this would involve doing two things.
During prediction the seed provided during fit isn't available, so we'd need this to be stored in the model (everywhere) - maybe there's a decorator or something that we can add for this?
Next, we need to make the data seed available. This should be added to the DataTuple/TestTuple.

I don't think that either of these are particularly crazy, but they are fairly large jobs given the prevalence of the classes affected.

Reply:
Okay, I see. For another time then.